### ISSUE TITLE
Supplementaries planters won't work with Cinderscapes installed [Fabric 1.20.1]

### ISSUE THREAD

**Issue by JadeiteWren** [TerraformersMC/Cinderscapes#166]
So I'm trying to use Cinderscapes in a mod pack (Fabric 1.20.1), but it's preventing me from placing flowers in Supplementaries planters? I downgraded but it doesn't fix the issue.

When I toggle off Cinderscapes the planter lets me place flowers in it again. Even if I do a fresh install with just Amendments, Cinderscapes, Fabric API, Moonlight Lib, and Supplementaries, it still won't work.

![Screenshot_20240910_085605_Discord](https://github.com/user-attachments/assets/8c492905-a330-45ab-9e1a-834f343d91f3)

**Comment by gniftygnome** [comment]
Sorry I've not responded before now, but every time I see this issue I just kind of scratch my head.  We add reasonable things to various tags, but we aren't "breaking" any tags I'm aware of.  To look into this I'd need more information about what is supposedly going on...

**Comment by JadeiteWren** [comment]
You can't put any plants or flowers in the supplementaries mod planters. They won't place.

**Comment by gniftygnome** [comment]
Yes, I understood that from the original description, but it tells me nothing about what is going wrong.  I do not have time to go trace a problem in the Supplementaries source with this little information about what is going on.  I'm skeptical about this being a tag issue.  It should be far easier for MehVahdJukaar to trace down what is going wrong here than it is for me since the problem is fundamentally in Supplementaries regardless of whether Cinderscapes is triggering it somehow.  If somebody can give me a decent technical explanation of what is actually going wrong in the Supplementaries source, I can probably work out what is wrong with Cinderscapes, but I have not seen any technical explanation yet.  Otherwise this will sit until I have a lot of free time to burn, which is a rare occurrence to be sure.
